test/data/css/customProperties.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <style>
+      .test-custom-props {
+          --prop1: val1;
+          --prop2:val2;
+          --prop3:val3 ;
+          --prop4: "val4" ;
+      }
+    </style>
+  </head>
+  <body>
+    <div class="test-custom-props"></div>
+    <script src="../../jquery.js"></script>
+    <script src="../iframeTest.js"></script>
+    <script>
+      startIframeTest( jQuery( ".test-custom-props" ) );
```

Comment:
I guess it could, although you could say the same about this global css file what you wrote here:
https://github.com/jquery/jquery/pull/3199/files/641c1a64d281a7cea5edb9f8de248b2160884b2d#r70155287
:)

Although that CSS file is still quite short.